s3am/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:12.04
+
+MAINTAINER Frank Austin Nothaft
+MAINTAINER CJ Ketchum
+
+# install necessary dependencies
+RUN apt-get update && \
+    apt-get install -y python-dev \
+            gcc \
+            make \
+            libcurl4-openssl-dev \
+            python-virtualenv
+
+# create virtualenv and install s3am
+RUN virtualenv /opt/s3am
+RUN /opt/s3am/bin/pip install s3am==2.0
+
+# add wrapper script
+WORKDIR /opt/
+ADD wrapper.sh /opt/wrapper.sh
+
+ENTRYPOINT ["bash", "+x", "/opt/wrapper.sh"]
```

Comment:
Lets omit the wrapper and make s3am the entry point. The perm fix is taken care of by docker_call now.